encoding.bs

```diff
@@ -877,39 +877,15 @@ different format here, to be able to represent ranges.)
 fallback encoding <var>encoding</var>, run these steps:
 
 <ol>
- <li><p>Let <var>buffer</var> be an empty byte sequence.
-
- <li><p>Let <var>BOMSeen</var> be false.
-
- <li><p><a>Read</a> bytes from <var>stream</var>
- into <var>buffer</var> until either <var>buffer</var> contains three bytes or
- <a>read</a> returns <a>end-of-stream</a>.
-
- <li>
-  <p>For each of the rows in the table below, starting with the first
-  one and going down, if the first bytes of <var>buffer</var> match
-  all the bytes given in the first column, then set <var>encoding</var>
-  to the <a for=/>encoding</a> given in the cell in the second column of
-  that row and <var>BOMSeen</var> to true.
-
-  <table>
-   <tbody><tr><th>Byte order mark<th>Encoding
-   <tr><td>0xEF 0xBB 0xBF<td><a>UTF-8</a>
-   <tr><td>0xFE 0xFF<td><a>UTF-16BE</a>
-   <tr><td>0xFF 0xFE<td><a>UTF-16LE</a>
-  </table>
+ <li><p><a>BOM sniff</a> <var>stream</var>. If the result is not null, set <var>encoding</var> to
```

Comment:
Let's make this something like  Let `BOMEncoding` be the result of BOM sniffing stream and then drop the BOMSeen variable. And override `encoding` in a subsequent step.

Reply:
Also, the p will need to be on its own line as the li has two children.